lib_trades/data/gm4_trades-1.1/functions/resolve_load.mcfunction

```diff
@@ -0,0 +1,4 @@
+execute if score gm4_trades load.status matches 1 if score gm4_trades_minor load.status matches 1 run function gm4_trades-1.1:load
+
+execute unless score gm4_trades load.status matches 1 run schedule clear gm4_trades-1.1:main
+execute if score gm4_trades load.status matches 1 unless score gm4_trades_minor load.status matches 1 run schedule clear gm4_trades-1.1:main
```

Comment:
Why is this commit part of this PR? Why does the version need to be bumped?

Reply:
lib_trades was changed to not change wandering traders with the `smithed.entity` tag. Since a change was made, the version needs to be bumped.